discovery-provider/src/api/v1/tracks.py

```diff
@@ -425,6 +428,13 @@ def get(self, track_id):
         if not creator_nodes or not track:
             abort_not_found(track_id, ns)
 
+        track_cid = track["track_cid"]
```

Comment:
Haven't been part of these convos but just double-checking that we've thought about rolling this change out and not breaking things while we're in the intermediate state, with nodes running different versions.

Reply:
good callout. but yeah we should be okay. overwhelming majority of tracks have cids. the ones that don't are in a weird state anyway